apis/io-engine/protobuf/v1/replica.proto

```diff
@@ -78,11 +79,23 @@ message UnshareReplicaRequest {
   string uuid = 1;  // uuid of the replica
 }
 
+// Resize(expand or shrink) a replica. Only valid for normal lvol
+// replicas, and not lvols that are a snapshot.
+message ResizeReplicaRequest {
+  string uuid = 1;  // uuid of the replica.
+  uint64 requested_size = 2; // indicates the requested new size of replica.
+}
+
 // List of replicas and their properties.
 message ListReplicasResponse {
   repeated Replica replicas = 1;  // list of the replicas
 }
 
+// Response containing updated replica information and result.
+message ResizeReplicaResponse {
```

Comment:
Could we return Replica directly?
See Share/Unshare

Reply:
> Could we return Replica directly? See Share/Unshare

I had two minds about doing that. I find this wrapped response more intuitive. In fact almost all of the response messages in Nexus protobuf follow this way.

Reply:
Yeah I know about the nexus that's I suggest, because we have to unwrap nexus on the client side when handling response.
Also every rpc other than list returns Replica so it'd be good to stay consistent as well?

Reply:
Alright, updated to return `Replica` directly. PTAL